schema/schema.sql

```diff
@@ -0,0 +1,19 @@
+-- games table
+CREATE TABLE IF NOT EXISTS games (
+    id INTEGER PRIMARY KEY,
+    name TEXT,
+    genre TEXT,
+    author TEXT,
```

Comment:
the author is a foreign key to this table so you can say: `author_id INTEGER,
    CONSTRAINT fk_author FOREIGN KEY (author_id) REFERENCES author(id) ON DELETE CASCADE`

Reply:
Thanks a lot, @myelin0!